apps/api/src/applicant/external-api.service.ts

```diff
@@ -450,6 +450,7 @@ export class ExternalAPIService {
         assigned_to: { id: string; name: string }[] | undefined;
         registration_date: string;
         applicant_id: string;
+        ats1_id: number;
```

Comment:
Probably missing some context on this one, but you have this listed as string in other uses.